src/components/TextInput/TextInput.tsx

```diff
@@ -1,18 +1,29 @@
 import React, { forwardRef } from 'react';
 import cx from 'classnames';
 
+/**
+ * The spellcheck attribute is an enumerated one and not a Boolean.
+ * This means that the explicit usage of one of the values true or false is mandatory.
```

Comment:
This reads as if you intend this to be a mandatory prop. Probably better written as "This means that only one of the string values 'true' or 'false' may be used."